src/app/types/Connector.ts

```diff
@@ -0,0 +1,51 @@
+export const CONNECTOR_ALL_TYPES_MAP =
```

Comment:
This is more a UI model than a type like logs.model.ts file.
Move this definition in the charging-stations.model.ts file.

Reply:
will commit change ✅